tests/fixtures/common/collections/ansible_collections/testorg/coll_1/plugins/lookup/lookup_1.py

```diff
@@ -2,6 +2,7 @@
 
 from __future__ import absolute_import, division, print_function
 
+
```

Comment:
Since you've excluded the entire directory and isort no longer manages this file, do you want to also revert this change across the directory to exclude it from the PR? Or do you want to still keep it?

Reply:
It's a good point, since isort is ignoring the entire directory, changes to these files should be left out of the PR.  thanks, keen eye :)